### PR TITLE
PAYARA-3451 Managed Scheduled Executor Service doesn't execute tasks from versioned applications (Fix-2)

### PR DESCRIPTION
**Pull request by jGauravGupta**

### CODE REVIEW

**Comment by jGauravGupta** [comment]
jenkins test please

File: appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/runtime/ContextSetupProviderImpl.java [hunk 1]
```diff
@@ -344,6 +346,19 @@ private boolean isApplicationEnabled(String appId) {
         }
         return result;
     }
+    
+    private boolean isApplicationAvailable(String appId) {
+        boolean result = false;
+        if (appId != null) {
+            Application app = applications.getApplication(appId);
+            if (app != null) {
+                result = true;
```
Comment (MarkWareham) [inline appserver/concurrent/concurrent-impl/src/main/java/org/glassfish/concurrent/runtime/ContextSetupProviderImpl.java]: Could just directly return here

**Comment by jGauravGupta** [comment]
This PR is not intended to be merge,  just to showcase the alternate solution of #3693 .

**Comment by Pandrex247** [comment]
We can close the PR and the diff will still be visible :)

**Pull request closed without merging**